packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/site-resolver/plugins/default.ts

```diff
@@ -0,0 +1,26 @@
+import { SiteInfo, tryParseEnvValue } from '@sitecore-jss/sitecore-jss-nextjs/middleware';
+import config from 'temp/config';
+import { SiteResolverPlugin } from '..';
+
+// Resolving from env variable, but it can be expanded or change in future if needed.
+const pointOfSale = tryParseEnvValue<Record<string, string>>(
+  process.env.NEXT_PUBLIC_CDP_POINTOFSALE,
+  {}
```

Comment:
Why not set the default value here instead of below? Makes more sense to have all the parsing logic in one place.
```suggestion
  { [config.defaultLanguage]: process.env.NEXT_PUBLIC_CDP_POINTOFSALE || '' }
```